lib/screens/search.dart

```diff
@@ -71,15 +71,7 @@ class _ProductSearchState extends State<ProductSearch> {
                 padding: EdgeInsets.only(left: 15),
                 child: TextField(
                   controller: _controller,
-                  onChanged: (value) {
-                    setState(() {
-                      slug = value;
-                      isSearched = true;
-                      searchProducts = [];
-                      currentPage = 1;
-                      searchProduct();
-                    });
-                  },
+                  onChanged: (value) {},
```

Comment:
The value that user enters should be saved in the "slug" variable and then sent with the API call.

Reply:
Please set this currently same search results are shown always as you are sending just a space in the API.